.github/workflows/ci.yml

```diff
@@ -46,22 +46,28 @@ jobs:
         run: just ext docker-run
 
       - name: Build
-        run: docker exec pgai-ext just build
+        run: docker exec 
+          -w /pgai/projects/extension
```

Comment:
Specifying `-w` for all usages feels like a mistake?

Reply:
yeah, i'm not sure it's necessary. I'm being paranoid.

Reply:
yep, now that i set the workdir in the dockerfile, it should work without this noise